### PR TITLE
[INLONG-3556][Dashboard] Add Iceberg params to adapt the Manager module

### LINKED ISSUE #3556: [Feature][Dashboard] Add Iceberg params to adapt the Manager module
### Description

Add Iceberg params to adapt the Manager module.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Title Name: [INLONG-3556][Dashboard] Add Iceberg params to adapt the Manager module

Fixes #3556 

### Motivation

Add Iceberg params to adapt the Manager module.

### Verifying this change

- [X] Make sure that the change passes the CI checks.

### CODE REVIEW

File: inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx [hunk 1]
```diff
@@ -186,11 +187,27 @@ const Comp: React.FC<DetailModalProps> = ({
...
+          },
+        ],
         props: {
           disabled: !!id,
         },
+      },
+      {
+        name: 'description',
+        type: 'textarea',
+        label: i18n.t('components.AccessHelper.StorageMetaData.Description'),
```
Comment (leezng) [inline inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx:205]: You can use `t('')` directly here, no need for `i18n.t`

File: inlong-dashboard/src/components/MetaData/StorageIceberg.tsx [hunk 2]
```diff
@@ -128,27 +130,96 @@ const getForm: GetStorageFormFieldsType = (
...
+      initialValue: 1,
+      tooltip: i18n.t('components.AccessHelper.StorageMetaData.Hive.EnableCreateResourceHelp'),
       props: {
         disabled: isEdit && [110, 130].includes(currentValues?.status),
         options: [
           {
-            label: 'Iceberg',
-            value: 'Iceberg',
+            label: i18n.t('basic.Yes'),
+            value: 0,
```
Comment (leezng) [inline inlong-dashboard/src/components/MetaData/StorageIceberg.tsx:144]: Yes = 0 ?

File: inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx [hunk 3]
```diff
@@ -186,11 +187,27 @@ const Comp: React.FC<DetailModalProps> = ({
...
+        rules: [{ required: false }],
```
Comment (leezng) [inline inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx:206]: This is not required, no need to set rules.

**Review by leezng — CHANGES_REQUESTED** [review]

File: inlong-dashboard/src/components/MetaData/StorageIceberg.tsx (hunk 2 above)
Comment (healchow) [reply inlong-dashboard/src/components/MetaData/StorageIceberg.tsx:144]: changed to 1.

File: inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx (hunk 3 above)
Comment (healchow) [reply inlong-dashboard/src/components/AccessHelper/DataStorageEditor/DetailModal.tsx:206]: OK, removed.

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**